### PR TITLE
Add Firewall Analytics documentation

### PR DESCRIPTION
**Pull request by pedrosousa**
Migrates the following Support KB to Dev Docs:
https://support.cloudflare.com/hc/en-us/articles/360024520152-Understanding-Cloudflare-Firewall-Analytics

Addresses https://jira.cfops.it/browse/PCX-2766.

### CODE REVIEW

**Comment by pedrosousa** [comment]
cc @zeinjaber

File: products/waf/src/content/analytics/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Users on a Free plan can view summarized Firewall events by date in the **Activity log**. Customers on paid plans have access to additional graphs and dashboards that summarize the most relevant information about the current behavior of Cloudflare's security features and any recent threats against your zone.
+
+## Main features
+
+* **Events summary**: Provides the number of Firewall events on traffic during the selected time period, grouped according to the selected dimension (for example, Action, Host, Country).
+* **Events by service**: Lists the Firewall activity per Cloudflare security feature (for example, WAF, Firewall Rules, API Shield).
+* **Top events by source**: Provides details of the traffic flagged or actioned by a Cloudflare security feature (for example, IP addresses, User Agents, Paths, Countries, Hosts, ASNs).
+* **Activity log**: Summarizes Firewall events by date to show the action taken and the applied Cloudflare security product.
+* **Denial-of-service attacks mitigated**: Counts automatically mitigated layer 4 attacks blocked by Cloudflare over the last 7 days.
```
Comment (marciocloudflare) [inline products/waf/src/content/analytics/index.md]:
```suggestion
* **Denial-of-service attacks mitigated**: Counts automatically mitigated Layer 4 attacks blocked by Cloudflare over the last seven days.
```

File: products/waf/src/content/analytics/paid-plans.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Events by service
+
+The **Events by service** section lists the Firewall activity per Cloudflare security feature (for example, **WAF**, **Firewall Rules**, **API Shield**).
+
+You can adjust the scope of Firewall Analytics to one of the displayed services by clicking **Filter** or **Exclude** when hovering the legend or by clicking the corresponding graph bar.
+
+## Top events by source
+
+In **Top events by source** you can find details of the traffic flagged or actioned by a Cloudflare security feature — for example, **IP Addresses**, **User Agents**, **Paths**, **Countries**, **Firewall Rules**.
```
Comment (marciocloudflare) [inline products/waf/src/content/analytics/paid-plans.md]: I think it should be something like `**Countries**, and  **Firewall Rules**` but I'm not sure as I don't know if these are all the options or some of the options. Either way, I think there's something missing before the last example.

File: products/waf/src/content/analytics/paid-plans.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## Layer 4 denial-of-service attack mitigation
+
+<Aside type="note">
+
+Only available on Enterprise plans.
+
+</Aside>
+
+In **Denial-of-service attacks mitigated** you have visibility over mitigated layer 4 denial-of-service SYN attacks towards your zones in the past 7 days.
```
Comment (marciocloudflare) [inline products/waf/src/content/analytics/paid-plans.md]:
```suggestion
In **Denial-of-service attacks mitigated** you have visibility over mitigated Layer 4 denial-of-service SYN attacks towards your zones in the past seven days.
```

**Review by marciocloudflare — APPROVED** [review]

File: products/waf/src/content/analytics/paid-plans.md (hunk 2 above)
Comment (pedrosousa) [reply products/waf/src/content/analytics/paid-plans.md]: In fact, these are just examples (it's not a comprehensive list), but I'll add the `and` nevertheless — we already say that they're examples.

Comment (pedrosousa) [inline products/waf/src/content/analytics/paid-plans.md]:
```suggestion
In **Top events by source** you can find details of the traffic flagged or actioned by a Cloudflare security feature — for example, **IP Addresses**, **User Agents**, **Paths**, **Countries**, and **Firewall Rules**.
```

File: products/waf/src/content/analytics/paid-plans.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+<FilterReportDuration/>
+
+## Create Firewall Rule from current filters
+
+<CreateFirewallRule/>
+
+## Events summary
+
+The **Events summary** section provides the number of Firewall events on traffic during the selected time period, grouped according to the selected dimension (for example, **Action**, **Host**, **Country**, **ASN**).
```
Comment (pedrosousa) [inline products/waf/src/content/analytics/paid-plans.md]:
```suggestion
The **Events summary** section provides the number of Firewall events on traffic during the selected time period, grouped according to the selected dimension (for example, **Action**, **Host**, **Country**, or **ASN**).
```

**Pull request merged**